app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -0,0 +1,136 @@
+  - Use Environment variables like `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+  - Use profile and credential file provided by the environment variable `AWS_PROFILE` and `AWS_SHARED_CREDENTIALS_FILE`.
+  - Use [ECS Task IAM role](https://docs.aws.amazon.com/AmazonECS/latest/userguide/task-iam-roles.html) automatically when the Kong instance is running inside an ECS container
+  - Use [IAM role defined by serviceaccount](https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html) inside EKS pod
+  - Use [EC2 IAM role](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) inside EC2 environment
+
+{:.warning}
+> You **cannot** change the value of the environment variables that used to provide the AWS Credential after Kong instance boot up. The change will be ignored and **will not take effect** inside Kong instance.
+
+- **Ensure the IAM role you assinging to Kong has proper IAM policy**. Below is a simple example policy allows a user "john" to connect to a RDS instance "db-ABCDEFGHIJKL01234" by using the IAM database authentication:
+
+```json
+{
+   "Version": "2012-10-17",
+   "Statement": [
+      {
+         "Effect": "Allow",
+         "Action": [
+             "rds-db:connect"
+         ],
+         "Resource": [
+             "arn:aws:rds-db:us-east-2:1234567890:dbuser:db-ABCDEFGHIJKL01234/john"
+         ]
+      }
+   ]
+}
+```
+
+{:.note}
+> **Note:** In this example, `db-ABCDEFGHIJKL01234` is the resource ID of the RDS instance, which can be found in the "Configuration" page on the AWS Console. If you're using Aurora RDS with Aurora PostgreSQL engine, you can also use a cluster ID to grant permission to authenticate to the cluster, for example `cluster-ABCDEFGHIJKL01234`.
+
+For more details, please check the AWS documentation: [https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.IAMPolicy.html)
+
+- **Ensure you have the database account created in the RDS**. You need to create the database user and then grant them the `rds_iam` role as shown in the following example:
+
+```text
+# Connect to your RDS instance by using your master user/password
+$ psql "host=db-your-instance-1.ABCDEFGHIJ12345.us-east-1.rds.amazonaws.com port=5432 user=testmaster password=testpassword dbname=postgres"
```

Comment:
@hat do you think should we add also the `psql` step for granting privileges?

```
grant all privileges on database kong to kong_test
```

Reply:
I mentioned that `Don't forget to provision the database with creating the database and granting enough permissions to the database user you've just created! You can check the docs at [Using a database](/gateway/latest/install/linux/debian/#using-a-database) in our Installation Options(/gateway/latest/install/)`

So I think we don't need to mention it again here?